source/universal/components/Dashboard/DashboardContent.js

```diff
@@ -40,13 +49,17 @@ function indexTags(tags) {
   return colorMap;
 }
 
-export default ({stars, tags}) => {
+export default ({addTag, stars, tags}) => {
 
   const colorMap = indexTags(tags);
 
   return (
     <div className='dashboard'>
       <div className="dashboard__tags">
+        <div onClick={() => {
+          const name = prompt('enter the tag name');
```

Comment:
We should move this `prompt` into the action creator, i.e. `addTag` to make UI as simple as possible.